README.md

```diff
@@ -46,8 +46,9 @@ acf_add_local_field(array(
 
 ## About
 
-Version 1.0.4
+Version 1.0.5
 
 Written by Irvin Lim. If you encounter any issues, do open [one](https://github.com/irvinlim/acf-recaptcha/issues/new).
+WordPress wp_remote_post usage in WP Class written by Ramon Fincken (v 1.0.5)
```

Comment:
Would appreciate it if the Markdown was written properly! Single newlines are not displayed in Markdown.

Perhaps you can create a **Contributors** section? Thanks!